.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: erlang
+
+otp_release:
+    - 21.1.3
+    - 20.3.8
+
+install:
+   - wget -c https://github.com/erlang/rebar3/releases/download/3.7.5/rebar3 && chmod +x rebar3
```

Comment:
i'll need to re-verify anything above 3.6.2... 3.7.X broke many plugins and i want to make sure our `rebar3_erllamba` will not

Reply:
I think it is the same, but if needed, I can change it to 3.6.2.